### PR TITLE
feat!: v4.0

### PR DESCRIPTION
**Pull request by brivu**
This `PR` contains the changes required for release of `v4.0` of the `AWS ECS` Orb: 

1. Update variable names to conform with Orb Standardization Guide. 
2. Replace job `images` with the default executor 
3. Minor refactoring

### CODE REVIEW

File: src/commands/update_task_definition.yml [hunk 1]
```diff
@@ -69,18 +69,18 @@ steps:
       name: Retrieve previous task definition and prepare new task definition values
       command: <<include(scripts/get_prev_task.sh)>>
       environment:
-        ORB_EVAL_FAMILY: <<parameters.family>>
+        ORB_STR_FAMILY: <<parameters.family>>
         ORB_EVAL_CONTAINER_IMAGE_NAME_UPDATES: <<parameters.container_image_name_updates>>
         ORB_EVAL_CONTAINER_ENV_VAR_UPDATE: <<parameters.container_env_var_updates>>
```
Comment (EricRibeiro) [inline src/commands/update_task_definition.yml:74]: Rename to `ORB_STR` and change the relevant scripts accordingly.

File: src/commands/update_task_definition.yml [hunk 2]
```diff
@@ -69,18 +69,18 @@ steps:
...
         ECS_SCRIPT_UPDATE_CONTAINER_DEFS: <<include(scripts/update_container_defs.py)>>
         ECS_SCRIPT_GET_TASK_DFN_VAL: <<include(scripts/get_task_dfn_val.py)>>
-        ORB_EVAL_PROFILE_NAME: <<parameters.profile_name>>
+        ORB_STR_PROFILE_NAME: <<parameters.profile_name>>
         ORB_EVAL_PREVIOUS_REVISION_NUMBER: <<parameters.previous_revision_number>>
         ORB_EVAL_CONTAINER_SECRET_UPDATES: <<parameters.container_secret_updates>>
         ORB_EVAL_CONTAINER_DOCKER_LABEL_UPDATES: << parameters.container_docker_label_updates >>
```
Comment (EricRibeiro) [inline src/commands/update_task_definition.yml:80]: Same as above.

**Review by EricRibeiro — CHANGES_REQUESTED** [review]

File: src/scripts/get_prev_task.sh [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 
-# These variables are evaluated so the config file may contain and pass in environment variables to the parameters.
-ORB_EVAL_FAMILY="$(circleci env subst "$ORB_EVAL_FAMILY")"
-ORB_EVAL_CONTAINER_IMAGE_NAME_UPDATES="$(circleci env subst "$ORB_EVAL_CONTAINER_IMAGE_NAME_UPDATES")"
-ORB_EVAL_CONTAINER_ENV_VAR_UPDATE="$(circleci env subst "$ORB_EVAL_CONTAINER_ENV_VAR_UPDATE")"
-ORB_EVAL_PROFILE_NAME="$(circleci env subst "$ORB_EVAL_PROFILE_NAME")"
-ORB_EVAL_CONTAINER_SECRET_UPDATES="$(circleci env subst "$ORB_EVAL_CONTAINER_SECRET_UPDATES")"
-ORB_EVAL_CONTAINER_DOCKER_LABEL_UPDATES="$(circleci env subst "$ORB_EVAL_CONTAINER_DOCKER_LABEL_UPDATES")"
-ORB_EVAL_PREVIOUS_REVISION_NUMBER="$(circleci env subst "$ORB_EVAL_PREVIOUS_REVISION_NUMBER")"
+# These variables are STRuated so the config file may contain and pass in environment variables to the parameters.
```
Comment (EricRibeiro) [inline src/scripts/get_prev_task.sh:4]:
```suggestion
# These variables are evaluated so the config file may contain and pass in environment variables to the parameters.
```

**Review by EricRibeiro — CHANGES_REQUESTED** [review]

**Review by EricRibeiro — APPROVED** [review]

**Comment by mbeacom** [comment]
@brivu When might we expect version 4.x to be released?

**Pull request merged**